main.py

```diff
@@ -0,0 +1,113 @@
+import time
+import re
+import streamlit as st
+import utils
+from types import *
+
+password = 'ikun'
+tokens = [""] * 4
```

Comment:
我觉得 [""]*4 的话不太好，因为这个地方是给人用来填多个token用的（这个是我的问题，我readme没写好，我到时候会补充文档的
```suggestion
tokens = ["","",""]
```）

Reply:
OK, 原来是这个意思，我认为可以独立配置